.github/CODEOWNERS

```diff
@@ -4,4 +4,6 @@
 /x-pack/plugins/apm/  @elastic/apm-ui
 /x-pack/plugins/security/  @elastic/kibana-security
 /x-pack/plugins/spaces/  @elastic/kibana-security
+/x-pack/plugins/beats_management/  @elastic/beats
```

Comment:
Awesome, welcome to CODEOWNERS!

**nit**: sorry to be so picky, but would you mind reordering to keep the list alphabetical? I know that GitHub says that order matters when applying rules, but I don't expect to run into any ordering discrepancies for the x-pack plugin set

Reply:
NP, done